notebooks/203-meter-reader/203-meter-reader-digital.ipynb

```diff
@@ -0,0 +1,1087 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "17d59075-288a-46d0-b049-ed656fa7cf55",
+   "metadata": {},
+   "source": [
+    "### Get Input Image"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "3bfb78ff-b376-4777-bce3-b995c079f13b",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "# Download images\n",
+    "IMG_URL = \"https://user-images.githubusercontent.com/83450930/236680146-5751e291-d509-4d71-a2cb-bfbf35609051.jpg\"\n",
+    "IMG_FILE_NAME = IMG_URL.split(\"/\")[-1]\n",
+    "utils.download_file(IMG_URL, show_progress=False)\n",
+    "\n",
+    "# Read image\n",
+    "img = cv2.imread(IMG_FILE_NAME)\n",
+    "\n",
+    "# Show input image\n",
+    "plt.imshow(img)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "c0098067-45d3-42fe-b77f-4a4b62a34d9a",
+   "metadata": {},
+   "source": [
+    "### Preprocessing the input image\n",
+    "\n",
+    "Cut the screen part and use affine transformations to normalize skewed images."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "4d4b15cf-d519-4aa7-8dca-3e36e337f17c",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "# affine transformations to normalize skewed images\n",
+    "img = pre_processing(img, POINTS, DESIGN_SHAPE)\n",
+    "\n",
+    "# The screen part is cropped and corrected\n",
+    "plt.imshow(img)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "4b65d723-2b73-4d6f-9a02-f45c6e1f22e5",
+   "metadata": {
+    "tags": []
+   },
+   "source": [
+    "### Select the Region to be recognized"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "82571a0d-4779-41a3-b547-803da14b9b01",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "# features and layout information\n",
+    "DESIGN_LAYOUT = {'Info_Probe':[14, 36, 410, 135],  # feature_name, xmin, ymin, xmax, ymax\n",
+    "                 'Freq_Set':[5, 290, 544, 406], \n",
+    "                 'Val_Total':[52, 419, 1256, 741], \n",
+    "                 'Val_X':[19, 774, 433, 882], \n",
+    "                 'Val_Y':[433, 773, 874, 884], \n",
+    "                 'Val_Z':[873, 773, 1276, 883], \n",
+    "                 'Unit':[1064, 291, 1295, 403], \n",
+    "                 'Field':[5, 913, 243, 998]}"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "f1d0eabc-d187-4d84-963d-cda676daff89",
+   "metadata": {},
+   "source": [
+    "### Crop the element fields and recognition"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "0504e2d5-7fd0-47e1-8500-10f67ebcf96b",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "# the input of recognition should be image, DESIGN information, compiled_model\n",
+    "def main_for_field_strength_meter(img, DESIGN_LAYOUT, RESULT_TEMP, rec_compiled_model, rec_output_layer, text_decoder):\n",
```

Comment:
Since most of codes of 2 main function can be reusable except some special condition, I will highly recommend you to combine them into one, and put it in section 2.